crates/runtime/src/flight/flightsql/get_schemas.rs

```diff
@@ -0,0 +1,63 @@
+use arrow_flight::{
+    flight_service_server::FlightService,
+    sql::{self, ProstMessageExt},
+    FlightDescriptor, FlightEndpoint, FlightInfo, Ticket,
+};
+use prost::Message;
+use tonic::{Request, Response, Status};
+
+use crate::flight::{record_batches_to_flight_stream, to_tonic_err, Service};
+
+/// Get a `FlightInfo` for listing schemas.
+pub(crate) fn get_flight_info(
+    query: &sql::CommandGetDbSchemas,
+    request: Request<FlightDescriptor>,
+) -> Response<FlightInfo> {
+    tracing::trace!("get_flight_info");
+    let fd = request.into_inner();
+
+    let endpoint = FlightEndpoint::new().with_ticket(Ticket {
+        ticket: query.as_any().encode_to_vec().into(),
+    });
+
+    let info = FlightInfo::new()
+        .with_endpoint(endpoint)
+        .with_descriptor(fd);
+
+    Response::new(info)
+}
+
+pub(crate) async fn do_get(
+    flight_svc: &Service,
+    query: sql::CommandGetDbSchemas,
+) -> Result<Response<<Service as FlightService>::DoGetStream>, Status> {
+    let catalog = &query.catalog;
+    tracing::trace!("do_get: {query:?}");
```

Comment:
Is tracing like this able to delineate a `do_get` for a `CommandGetDbSchemas` against all other types of `do_get`s?

Reply:
Yeah, the module name is part of the output.